mvn test failed - XmlParserTest.testXmlParser - Connect Connection timed out

**Jetty version**
9.4.31.v20200723
**Java version**
1.8.0_191-b12
**OS type/version**
centos 8.0
**Description**
The error of executing mvn test after compilation is as follows：
> >>
```
[INFO] Running org.eclipse.jetty.xml.XmlAppendableTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.005 s - in org.eclipse.jetty.xml.XmlAppendableTest
[WARNING] Corrupted STDOUT by directly writing to native stream in forked JVM 1. See FAQ web page and the dump file /tmp/root/spack-stage/spack-stage-jetty-project-9.4.31.v20200723-7n6dmzqth4zyevt7ri74lgakat2xxdbe/spack-src/jetty-xml/target/surefire-reports/2021-02-24T16-47-18_345-jvmRun1.dumpstream
[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   XmlParserTest.testXmlParser:40 » Connect Connection timed out (Connection time...
[INFO]
[ERROR] Tests run: 95, Failures: 0, Errors: 1, Skipped: 1
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Jetty :: Project 9.4.31.v20200723:
[INFO]
[INFO] Jetty :: Build Resources ........................... SUCCESS [  0.488 s]
[INFO] Jetty :: Project ................................... SUCCESS [  3.048 s]
[INFO] Jetty :: Utilities ................................. SUCCESS [02:23 min]
[INFO] Jetty :: IO Utility ................................ SUCCESS [01:06 min]
[INFO] Jetty :: Http Utility .............................. SUCCESS [02:27 min]
[INFO] Jetty :: XML utilities ............................. FAILURE [02:18 min]
[INFO] Jetty :: JMX Management ............................ SKIPPED
[INFO] Jetty :: Utilities :: Ajax(JSON) ................... SKIPPED
```
Can you give me some help?

--- a/jetty-xml/src/test/java/org/eclipse/jetty/xml/XmlParserTest.java
+++ b/jetty-xml/src/test/java/org/eclipse/jetty/xml/XmlParserTest.java
@@ -33,6 +33,8 @@ public class XmlParserTest
 
         URL configURL = XmlConfiguration.class.getClassLoader().getResource("org/eclipse/jetty/xml/configure_9_3.dtd");
         parser.redirectEntity("configure.dtd", configURL);
+        parser.redirectEntity("configure_9_3.dtd", configURL);
+        //parser.redirectEntity("http://www.eclipse.org/jetty/configure_9_3.dtd", configURL);
         parser.redirectEntity("http://jetty.eclipse.org/configure.dtd", configURL);
         parser.redirectEntity("-//Mort Bay Consulting//DTD Configure//EN", configURL);
 

